--- khal-0.9.9/setup.py~	2018-05-26 19:53:19.000000000 +0000
+++ khal-0.9.9/setup.py	2018-09-26 14:37:17.032936084 +0000
@@ -15,7 +15,7 @@
     'urwid',
     'pyxdg',
     'pytz',
-    'python-dateutil<=2.6.1',
+    'python-dateutil',
     'configobj',
     # https://github.com/untitaker/python-atomicwrites/commit/4d12f23227b6a944ab1d99c507a69fdbc7c9ed6d  # noqa
     'atomicwrites>=0.1.7',
--- khal-0.9.9/khal.egg-info/requires.txt~	2018-05-26 20:12:15.000000000 +0000
+++ khal-0.9.9/khal.egg-info/requires.txt	2018-09-26 14:37:28.050935930 +0000
@@ -3,7 +3,7 @@
 urwid
 pyxdg
 pytz
-python-dateutil<=2.6.1
+python-dateutil
 configobj
 atomicwrites>=0.1.7
 tzlocal>=1.0
